static/app/views/issueList/overview.tsx

```diff
@@ -489,9 +489,9 @@ class IssueListOverview extends React.Component<Props, State> {
               })),
             };
           },
-          error: err => {
+          error: () => {
             this.setState({
-              error: parseApiError(err),
+              issuesLoading: false,
```

Comment:
Should the error be discarded here?

Reply:
yeah i don't think we need to set anything. maybe remove the queryCounts instead since it's no longer valid? might have to check if its cancelled but i would hope error isn't thrown on cancel